content/docs/rendering-elements.md

```diff
@@ -1,75 +1,74 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: Me-render Elemen
 permalink: docs/rendering-elements.html
 redirect_from:
   - "docs/displaying-data.html"
 prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+Elemen adalah blok pembangun terkecil dalam aplikasi React
 
-An element describes what you want to see on the screen:
+Sebuah elemen menggambarkan apa yang ingin anda lihat dalam layar:
 
 ```js
 const element = <h1>Hello, world</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+Tidak seperti elemen DOM, elemen pada React merupakan objek kosong, dan mudah dibuat. React DOM mengurus pembaharuan pada DOM untuk menyesuaikan dengan elemen pada React.
 
->**Note:**
+>**Catatan:**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+>Anda mungkin akan sedikit bingung antara elemen dengan konsep yang lebih dikenali yaitu "komponen". Kami akan memperkenalkan komponen di [bagian berikutnya](/docs/components-and-props.html). Elemen adalah apa yang "membuat" komponen, dan kami menyarankan anda untuk membaca bagian ini sebelum melompat ke bagian berikutnya.
+## Me-*render* elemen ke dalam DOM {#rendering-an-element-into-the-dom}
 
-## Rendering an Element into the DOM {#rendering-an-element-into-the-dom}
-
-Let's say there is a `<div>` somewhere in your HTML file:
+Katakanlah terdapat sebuah `<div>` di suatu tempat di *file* HTML anda:
```

Comment:
You can also say *Sebagai contoh* instead of *Katakanlah* when the context is trying to explain something with an example.